src/en-us.strings.js

```diff
@@ -190,5 +190,19 @@ export default {
 	'pdfReader.convertToHighlight': 'Convert to Highlight',
 	'pdfReader.convertToUnderline': 'Convert to Underline',
 	'pdfReader.size': 'Size',
-	'pdfReader.merge': 'Merge'
+	'pdfReader.merge': 'Merge',
+    'pdfReader.a11yAnnotationModifierMac': 'Option',
+    'pdfReader.a11yAnnotationModifier': 'Alt',
+    'pdfReader.a11yAnnotationModifierControl': 'Control',
+    'pdfReader.a11yTextualAnnotationFindInDocumentInstruction': 'To turn a search result into a highlight or underline annotation, press',
+    'pdfReader.a11yTextualAnnotationInstruction': 'To annotate text via the keyboard, first use “Find in Document” to locate the phrase. Then, to turn the search result into an annotation, press Control -',
+    'pdfReader.a11yAnnotationInstruction': 'To add this annotation to the document, focus the document and press Control -',
+    'pdfReader.a11yAnnotationNotSupported': 'This annotation type cannot be created via the keyboard.',
+    'pdfReader.a11yMoveAnnotation': 'Use the arrow keys to move the annotation.',
+    'pdfReader.a11yEditTextAnnotation': 'To move the end of the text annotation, use the left/right arrow keys while holding Shift. To move the start of the annotation, use the arrow keys while holding Shift -',
+    'pdfReader.a11yResizeAnnotation': 'To resize the annotation, use the arrow keys while holding Shift.',
+    'pdfReader.a11yAnnotationSelected': 'selected.',
```

Comment:
These partial fragments don't work. We can do it if we really need to for keyboard shortcuts, where we're just building up a string, but we can't assume a given sentence structure for localized strings.

So we'd have to use placeholders here or just have separate strings with hard-coded annotation types.

Reply:
Assuming we really need to say the annotation type, separate strings are probably the safest.

Reply:
The thing that goes before "selected" or "has been created" is the annotation type. In english, it becomes "Note annotation selected". Would that still cause issues? Would it be better to do something like "Selected: " and "Created: ", followed by annotation type? Like "Selected: Note annotation"?

Reply:
Just use separate strings. A fragment like that could easily not work in a given language, there could be gender differences, and at the very least it could totally confuse localizers — e.g., what is "selected."? I know it's antithetical to the programmer DRY ethos, but attempting to reuse strings is usually a bad idea when it comes to localization. (The main exception would be something like a button name — "and click {$button}".)

Reply:
Gotcha. Thanks for the explanation!

The last compromise I have in mind (without repetition that really doesn't sit well with me) is to keep initial sentences and just append the annotation type? It would say e.g. "Annotation has been selected. Note annotation.". A bit clunky but just as functional.

Reply:
Separate strings are OK! The strings will be clear and unclunky, people will localize them once, and we'll never need to think about them again (until we add more annotation types…).

Even in English, these only work by being semi-ungrammatical (or in "telegraphic style") and omitting certain words. Otherwise, "A [$annotationType] was selected" would be wrong for "underline annotation". Just use regular sentences!

Reply:
I mean I guess the only exception here would be if screen reader users would rather hear "Annotation selected: highlight" instead of "Highlight annotation selected" or "A highlight annotation was selected". You hear more of these strings, so you know better what the norm is.

Reply:
(So I'll maybe retract "Just use regular sentences" in this specific context, but the larger point stands — repetition is often the better option for localization.)

Reply:
Regular sentences work well! Screen readers do have their own ways to announce elements and their positioning, but as far as the announcements on the application level go, I believe full sentences are as clear as it can get. So I added `pdfReader.a11yAnnotationSelected.*` and `pdfReader.a11yAnnotationCreated.*` strings for each type. I will admit that it does feel a bit cleaned than combining strings on the fly.

Reply:
It occurs to me that telegraphic style avoids the problem of tense: "has been selected" or "is selected"? Present tense is maybe more normal for a screen reader? (I've mostly tested with VoiceOver: "You are in a…") So if we want to use full sentences, I'd probably go with "A highlight annotation is selected". For "created", not sure about "has been created" vs. "was created".

But while I don't actually know how other languages handle telegraphic style, we obviously use it elsewhere, so we totally can say "Highlight annotation selected" and "Highlight annotation created" here and avoid this problem (at least in English!) as long as we put them all in separate strings.

Reply:
Hm, yeah. Having gotten some sleep, I see the perfect tense is adding unnecessary complexity here, and "${annotation_type} annotation selected"/"${annotation_type} annotation created" would be better for localization and also a bit more concise. 

I think as far as how exactly we phrase the announcement (e.g. past vs perfect tense) doesn't really matter too much, as long as it properly informs the user what happened and where they are at. My impression is that screen readers approach these things in their own ways: e.g. Voiceover does announce "You are in ...", while NVDA just says "Grouping ..., button ...". But past tense with telegraphic style indeed seems most prevalent. For example, if you open firefox and type something (e.g. "zotero") in the url bar until autocomplete is suggested, you'll then hear "zotero.org selected instead", which is an alert announcement similar in nature to what we are doing here. (Note this example does not go through in voiceover).